async_downloads/cache.py

```diff
@@ -148,3 +148,22 @@ def cleanup_collection(collection_key):
     # cache.set(
     #     collection_key, [key for key in cache.get(collection_key, []) if cache.get(key) is not None]
     # )
+
+
+def set_error(download_key, error):
+    download = cache.get(download_key)
+    if not download:
+        return
+
+    logger.exception(
+        f"Download failed with error: {error}",
```

Comment:
```suggestion
        "Download failed via set_error: key: %s name: %s",
```
The following arguments are for insertion into the string - have a look at the other ones above.
I removed the error message in my suggestion for consistency with the logging above. It might be a good idea to add the error into the logs in all three cases in a separate PR.

Reply:
[9dd7d48](https://github.com/QuickRelease/django-async-downloads/pull/42/commits/9dd7d48a96b44968ce6e2ed37654c1db83d5a8af) 

I'll raise the error logging PR after this one is merged.